### PR TITLE
[stable/influxdb] Chart deprecation

### PR DESCRIPTION
**Pull request by naseemkullah**
Follows https://github.com/helm/charts/blob/master/PROCESSES.md cc @davidkarlsen @Aisuko 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x]  Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/influxdb/Chart.yaml [hunk 1]
```diff
@@ -1,20 +1,14 @@
 apiVersion: v1
 name: influxdb
-version: 4.3.1
+version: 4.3.2
 appVersion: 1.7.9
-description: Scalable datastore for metrics, events, and real-time analytics.
+deprecated: true
+description: DEPRECATED Scalable datastore for metrics, events, and real-time analytics.
```
Comment (davidkarlsen) [inline stable/influxdb/Chart.yaml:6]: is there a new home?

Comment (naseemkullah) [reply stable/influxdb/Chart.yaml:6]: Yep: https://github.com/helm/charts/blob/master/stable/influxdb/README.md#deprecated---influxdb

Already in the README from previous commit.

**Review by Aisuko — APPROVED** [review]
/approve

**Pull request merged**